BaseInterfaces/src/interfaces/array.jl

```diff
@@ -201,4 +201,4 @@ array_components = (;
 
 _wrappertype(A) = Base.typename(typeof(A)).wrapper
 
-@interface ArrayInterface AbstractArray array_components "Base Julia AbstractArray interface"
+@interface ArrayInterface <: IterationInterface{(:reverse,:indexing)} AbstractArray array_components "Base Julia AbstractArray interface"
```

Comment:
Interface definitions are becoming fairly long to put everything on a single line.
I remember we had discussed defining the documentation somewhere else and just giving the macro a `String` variable, but for some reason it didn't work?